### DISCUSSION TITLE
Repeat with label_values provide combinaisons rows since 15 days

### DISCUSSION THREAD

**Discussion by Yaume2** [grafana/grafana#50284] [Dashboards, Panels & Visualizations]
Hello,
I've got a dashboard with prometheus datasource. I use windows-exporter and the variable :
{
        "datasource": "prometheus",
        "definition": "label_values(windows_cpu_time_total, instance)",
        "hide": 0,
        "includeAll": true,
        "label": "Server",
        "multi": true,
        "name": "server",
        "options": [],
        "query": {
          "query": "label_values(windows_cpu_time_total, instance)",
          "refId": "prometheus-server-Variable-Query"
        }

This variable is used to repeat a row of panels. One row by server.
 {
      "collapsed": false,
      "gridPos": {
        "h": 1,
        "w": 24,
        "x": 0,
        "y": 0
      },
      "title": "$server",
      "repeat": "server",
      "type": "row"
    }
I've got 2 servers (for example) : server 1 and server 2.
Until last week, i saw 2 rows. One for server 1 and one for server 2.
Last week after stop and relaunch (it's a docker container with a provisioned dashboad). I saw the two rows but also a row "All" with value "server 1 + server 2". And for each refresh a new line "All" is added.
What is wrong please ?

![grafana-all](https://user-images.githubusercontent.com/18381447/172330516-2f46fe5c-7121-4b94-a62b-0a67db0fa7be.png)

**Comment by Yaume2** [comment]
Ok i found solution but not explanation.
I've removed a not used variable (generated by ui) : Intervall and it works.
I don't know why.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
